deploy/crds/crd-certificates.yaml

```diff
@@ -80,6 +80,12 @@ spec:
                         enum:
                           - DER
                           - CombinedPEM
+                cleanupPolicy:
+                  description: CleanupPolicy is when this field is set to `Delete`, the owner reference is always created on the Secret resource and the secret will be automatically removed when the certificate resource is deleted. When this field is set to `Never`, the owner reference is never created on the Secret resource and the secret will not be automatically removed when the certificate resource is deleted. If the value of this field is unset this field "inherits" the value of the flag `--enable-certificate-owner-ref`.
+                  type: string
+                  enum:
+                    - Never
+                    - Delete
```

Comment:
Should we make this 'OnDelete' instead of 'Delete'? Given 'never' is a reference to a point in time, OnDelete seems more appropriate to me as a phrase that implies when to cleanup (rather than 'Delete' now which describes an action to take)